src/main/cpp/blaze.cc

```diff
@@ -1007,6 +1007,11 @@ static DurationMillis ExtractData(const string &self_path,
           << "install base directory '" << install_base
           << "' could not be created. It exists but is not a directory.";
     }
+
+    if (startup_options.trust_install_base) {
```

Comment:
I am a little surprised to see this down here.

What's the expected behavior of this option when no install base exists? I would not assume that it should create one (but it does above).